vector/read.go

```diff
@@ -0,0 +1,432 @@
+package vector
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/netip"
+
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/pkg/nano"
+	"github.com/brimdata/zed/vng"
+	vngVector "github.com/brimdata/zed/vng/vector"
```

Comment:
We use lowercase package names.
```suggestion
	vngvector "github.com/brimdata/zed/vng/vector"
```